packages/sdk/src/WamNode.js

```diff
@@ -23,6 +23,22 @@ const WamEventRingBuffer = getWamEventRingBuffer();
  * @implements {IWamNode}
  */
 export default class WamNode extends AudioWorkletNode {
+	/**
+	 * Register scripts required for the processor. Must be called before constructor.
+	 * @param {BaseAudioContext} audioContext
+	 * @param {string} baseURL
+	 */
+	static async addModules(audioContext, baseURL) {
+		await audioContext.audioWorklet.addModule(`${baseURL}/../../sdk/src/RingBuffer.js`);
```

Comment:
```
We are losing the caching system which is normally there for addModules(fileUrl). If we import two same file (function string) it will cause problem.
```
In a situation where a host is loading multiple WAMs from various vendors and websites, I think this would already be broken, no?

Reply:
Yes let's merge this PR and sort out this issue there instead

Reply:
yeah that's fine, calling this is optional anyways